moveit_planners/stomp/examples/stomp_config_example.yaml

```diff
@@ -0,0 +1,60 @@
+stomp/manipulator:
```

Comment:
move this into the setup assistant template

Reply:
integrated this into the moveit setup assistant in my [commit here](https://github.com/ros-planning/moveit/pull/965/commits/94ea16bd2d8e4b4be40ed237e14217dcac5a3ef0)